test/test.js

```diff
@@ -102,6 +107,45 @@ test('Metrics', async t => {
   })
 })
 
+test('Logs', async t => {
+  await t.test('No logs', async t => {
+    const XDG_STATE_HOME = join(tmpdir(), randomUUID())
+    const { stdout } = await execa(
+      station,
+      ['logs'],
+      { env: { XDG_STATE_HOME } }
+    )
+    assert.strictEqual(stdout, '')
+  })
+  await t.test('With logs', async t => {
+    const XDG_STATE_HOME = join(tmpdir(), randomUUID())
+    await fs.mkdir(getPaths(XDG_STATE_HOME).moduleLogs, { recursive: true })
+    await fs.writeFile(getPaths(XDG_STATE_HOME).allLogs, '[date] beep boop\n')
+    const { stdout } = await execa(
+      station,
+      ['logs'],
+      { env: { XDG_STATE_HOME } }
+    )
+    assert.strictEqual(
+      stdout,
+      '[date] beep boop'
+    )
+  })
+
+  await t.test('Follow', async t => {
+    for (const flag of ['-f', '--follow']) {
+      const XDG_STATE_HOME = join(tmpdir(), randomUUID())
+      await fs.mkdir(getPaths(XDG_STATE_HOME).moduleLogs, { recursive: true })
+      const ps = execa(station, ['logs', flag], { env: { XDG_STATE_HOME } })
+      await Promise.all([
+        once(ps.stdout, 'data'),
+        fs.writeFile(getPaths(XDG_STATE_HOME).allLogs, '[date] beep boop\n')
+      ])
+      ps.kill()
+    }
```

Comment:
This test does not seem to assert anything. If that's intentional, then please add a comment to explain how this test works.